src/cmd/services/m3coordinator/ingest/write.go

```diff
@@ -124,13 +126,26 @@ func NewDownsamplerAndWriter(
 	workerPool xsync.PooledWorkerPool,
 	instrumentOpts instrument.Options,
 ) DownsamplerAndWriter {
-	scope := instrumentOpts.MetricsScope().SubScope("downsampler")
+	var (
+		scope = instrumentOpts.MetricsScope().SubScope("downsampler")
+
+		emptyAnnotationMetricName = "datapoints_with_empty_annotation"
+		typeTag                   = func(t string) map[string]string {
+			return map[string]string{"type": t}
```

Comment:
I'd really like to avoid further overloading of "type" term, this would only add to the confusion as the end goal of these new metrics is to track down the time series that are missing the actual "metrics type". Could we rename this label to "storage-policy" for example (as this value is actually derived from `StoragePolicy` value)?

Reply:
https://github.com/m3db/m3/pull/3553/commits/80e0205445535c5b780dcf63e623efc65cf98e27#diff-bd39ac14b90a7e77e48b860a55e1927d98fd3378a61676da3b11c8b4a7698958R133-R137